imports/ui/components/settings/Settings.jsx

```diff
@@ -158,6 +173,25 @@ class SettingsPage extends Component {
         )
     }
 
+
+    changePassword(event){
+        event.preventDefault();
+             if(this.state.newpassword != this.state.alterpassword){
+                 console.log(" you have entered the wrong password");
```

Comment:
make variables camel case and explicit each variable to a constant value.
E.g. `const {check1, check2} = this.state`